### PR TITLE
Fix CopyPopUpViewProducer

### PR DESCRIPTION
**Pull request by karreiro**
This PR fixes CopyPopUpViewProducer, in order to avoid code duplication (with `@FallbackImplementation`) and a potential memory leak. See more [here](https://github.com/uberfire/uberfire/pull/446).

### CODE REVIEW

**Comment by karreiro** [comment]
@paulovmr @mbarkley Could you please review this PR? :-)

**Comment by karreiro** [comment]
Jenkins, please retest this.

**Comment by paulovmr** [comment]
+1 to merge.

File: uberfire-extensions/uberfire-commons-editor/uberfire-commons-editor-client/src/main/java/org/uberfire/ext/editor/commons/client/file/CopyPopUpViewProducer.java [hunk 1]
```diff
@@ -16,34 +16,33 @@
...
-import javax.inject.Named;
 
 import org.jboss.errai.ioc.client.api.ManagedInstance;
+import org.uberfire.annotations.FallbackImplementation;
 import org.uberfire.commons.services.cdi.Startup;
 import org.uberfire.commons.services.cdi.StartupType;
 import org.uberfire.ext.editor.commons.client.file.popups.CopyPopUpPresenter;
 import org.uberfire.ext.editor.commons.client.file.popups.CopyPopUpView;
 
 @Startup(value = StartupType.BOOTSTRAP, priority = -1)
```
Comment (mbarkley) [inline uberfire-extensions/uberfire-commons-editor/uberfire-commons-editor-client/src/main/java/org/uberfire/ext/editor/commons/client/file/CopyPopUpViewProducer.java]: According to the javadoc for `Startup`, it is for server-side classes. We should probably remove it here.

**Comment by mbarkley** [comment]
Looks good. There's just one minor change I've suggested inline.

**Comment by karreiro** [comment]
Done :-) Thanks @paulovmr and @mbarkley

**Comment by mbarkley** [comment]
Another extremely minor thing, but you didn't remove the imports for `Startup` and `StartupType`.

**Comment by karreiro** [comment]
Ohh! Thanks @mbarkley

**Comment by mbarkley** [comment]
+1

**Comment by karreiro** [comment]
Jenkins, please retest this.

**Comment by csadilek** [comment]
@karreiro this is ready to merge but I think the commit message can be improved (some more details) i.e. 

Fix memory leak in CopyPopUpViewProducer
- Refactoring to reuse ```@FallbackImplementation``` instead of custom qualifier

**Comment by karreiro** [comment]
Thanks @csadilek

**Comment by ederign** [comment]
Thanks @karreiro and everyone involved in the review.

**Pull request merged**